velox/exec/tests/TableScanTest.cpp

```diff
@@ -271,6 +271,43 @@ TEST_F(TableScanTest, allColumns) {
   ASSERT_TRUE(it->second.dynamicFilterStats.empty());
 }
 
+TEST_F(TableScanTest, directBufferInputRawInputBytes) {
+  auto vectors = makeVectors(10, 1'000);
+  auto filePath = TempFilePath::create();
+  writeToFile(filePath->getPath(), vectors);
+  createDuckDbTable(vectors);
+
+  auto plan = PlanBuilder(pool_.get())
+                  .startTableScan()
+                  .outputType(rowType_)
+                  .endTableScan()
+                  .planNode();
+
+  std::unordered_map<std::string, std::string> config;
+  std::unordered_map<std::string, std::shared_ptr<Config>> connectorConfigs =
+      {};
+  auto queryCtx = std::make_shared<core::QueryCtx>(
+      executor_.get(),
+      core::QueryConfig(std::move(config)),
+      connectorConfigs,
+      nullptr);
+
+  auto task = AssertQueryBuilder(duckDbQueryRunner_)
+                  .plan(std::move(plan))
+                  .splits(makeHiveConnectorSplits({filePath}))
+                  .queryCtx(queryCtx)
+                  .assertResults("SELECT * FROM tmp");
+
+  // A quick sanity check for memory usage reporting. Check that peak total
+  // memory usage for the project node is > 0.
+  auto planStats = toPlanStats(task->taskStats());
+  auto scanNodeId = plan->id();
+  auto it = planStats.find(scanNodeId);
```

Comment:
can we combine current test to other test? https://github.com/facebookincubator/velox/pull/9801/files#diff-c62590f29a2bf22e50e276f2665e0a7666968880de73a6b4db745cbc0cfde6feR253

Reply:
@jinchengchenghh No. This is because all the other tests use `CachedBufferInput`. However, this unit test sets the cache to null when creating QueryCtx [here](https://github.com/facebookincubator/velox/pull/9801/files#diff-c62590f29a2bf22e50e276f2665e0a7666968880de73a6b4db745cbc0cfde6feR293), which allows it to reach the changes made in this PR to DirectBufferInput.